src/structures/Guild.js

```diff
@@ -419,7 +437,16 @@ class Guild {
    * @returns {Promise<GuildAuditLogs>}
    */
   fetchAuditLogs(options) {
-    return this.client.rest.methods.getGuildAuditLogs(this, options);
+    const _options = {
+      before: options.before,
+      after: options.after,
+      limit: options.limit,
+      user_id: this.client.resolver.resolveUserID(options.user),
+      action_type: options.type,
```

Comment:
Alright, I was just wondering cause I'm not used to leaving trailing commas, cause generally it crashes. Thanks for letting me know